articles/tutorials/get-user-information-with-unbounce-landing-pages.md

```diff
@@ -3,64 +3,67 @@
 ### Configuration on Auth0
 
 1. Create an account in [Auth0](https://auth0.com)
-2. Go to **Applications -> NEW**. Crete an HTML5 Application on Auth0 dashboard and go to Settings and take note of the Client ID and Domain.
+2. Go to **Clients -> + Create Client**. Crete a Single Page Application on the Auth0 dashboard, go to Settings and take note of the Client ID and Domain. Also, add the `callback URL` in both `Allowed Callback URLs` and `Allowed Origins (CORS)` (it should be your unbounce page URL. For example:`http://unbouncepages.com/changeit`).
```

Comment:
@Annyv2 Usually we use bold for field names, so I would replace `Allowed Callback URLs` and `Allowed Origins (CORS)` with **Allowed Callback URLs** and **Allowed Origins (CORS)**.